core/domain/email_manager_test.py

```diff
@@ -1249,6 +1250,90 @@ def test_that_suggestion_emails_are_correct(self):
                 feconf.EMAIL_INTENT_SUGGESTION_NOTIFICATION)
 
 
+class SubscriptionEmailTest(test_utils.GenericTestBase):
+    def setUp(self):
+        super(SubscriptionEmailTest, self).setUp()
+
+        self.signup(self.EDITOR_EMAIL, self.EDITOR_USERNAME)
+        self.editor_id = self.get_user_id_from_email(self.EDITOR_EMAIL)
+
+        self.signup(self.NEW_USER_EMAIL, self.NEW_USER_USERNAME)
+        self.new_user_id = self.get_user_id_from_email(self.NEW_USER_EMAIL)
+
+        self.exploration = self.save_new_default_exploration(
+            'A', self.editor_id, 'Title')
+        subscription_services.subscribe_to_creator(
+            self.new_user_id, self.editor_id)
+
+        self.can_send_emails_ctx = self.swap(
+            feconf, 'CAN_SEND_EMAILS', True)
+        self.can_send_subscription_email_ctx = self.swap(
+            feconf, 'CAN_SEND_SUBSCRIPTION_EMAILS', True)
+
+    def test_that_subscription_emails_are_correct(self):
```

Comment:
In email_manager.py I do raise an exception when `CAN_SEND_SUBSCRIPTION_EMAILS` or `CAN_SEND_EMAILS` is set to False. In email_manager_test.py I set both these values to true using the swap function. So could you please explain your suggestion a bit more?